packages/kbn-observable/README.md

```diff
@@ -0,0 +1,40 @@
+# kbn-observable
+
+This package contains a minimally future-[spec-compliant][spec] observable
+implementation. This package is only intended for internal use in the Kibana
+platform.
+
+## Background
+
+While the Kibana platform relies on RxJS observables internally, exposed
+observables should be minimally future-spec compliant. As there exists no other
```

Comment:
__question:__ `minimally` or `maximally`?

Reply:
Haha, yeah, that's not very clear 🙈 The intention was "the minimal implementation that does everything defined in the spec" (aka no other methods on the observables etc, e.g. like rxjs that _does_ follow the spec, but implements tons of helpers in addition). I'll re-word.